src/index.html

```diff
@@ -22,6 +22,431 @@
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="rating__desc">Reviews: 5</div>
+          </div>
+
+          <div class="card__price">
+            <div class="card__price-label">Price:</div>
+            <div class="card__price-value">$2,199</div>
+          </div>
+
+          <a
+            class="card__link"
+            href="#"
+            data-qa="hover"
+          >
+            Buy
+          </a>
+        </div>
+
+        <div class="card">
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+          />
+
+          <h3 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h3>
+
+          <div class="card__code">
+            <span class="card__code-label">Product code:</span>
+            <span class="card__code-value">195434</span>
+          </div>
+
+          <div class="card__rating rating">
+            <div class="rating__stars stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="rating__desc">Reviews: 5</div>
+          </div>
+
+          <div class="card__price">
+            <div class="card__price-label">Price:</div>
+            <div class="card__price-value">$2,199</div>
+          </div>
+
+          <a
+            class="card__link"
+            href="#"
+            data-qa="hover"
+          >
+            Buy
+          </a>
+        </div>
+
+        <div class="card">
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+          />
+
+          <h3 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h3>
+
+          <div class="card__code">
+            <span class="card__code-label">Product code:</span>
+            <span class="card__code-value">195434</span>
+          </div>
+
+          <div class="card__rating rating">
+            <div class="rating__stars stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="rating__desc">Reviews: 5</div>
+          </div>
+
+          <div class="card__price">
+            <div class="card__price-label">Price:</div>
+            <div class="card__price-value">$2,199</div>
+          </div>
+
+          <a
+            class="card__link"
+            href="#"
+            data-qa="hover"
```

Comment:
The `data-qa` attribute here is 'hover'. Consider using a consistent naming convention for `data-qa` attributes across similar elements.